credentials/apps/credentials/models.py

```diff
@@ -204,6 +204,11 @@ class CourseCertificate(AbstractCertificate):
     .. no_pii: This model has no PII.
     """
 
+    # We require signatories on program certificates, However, on course certificate configurations,
+    # not all sites will care. The certificates themselves don't render in the credentials IDA,
+    # and this is cloned data from the system of record.
```

Comment:
The comment below is kinda hard to parse. My suggestion is really just a restructuring of what you wrote if it helps:

```suggestion
    # Not all sites will require signatories in course certificate configurations, but signatories are required in program certificates configurations.
    # This is cloned data from the system of record, as the course certificates don't render in the Credentials IDA.
```

Reply:
this makes sense to me, I committed your suggestion.